Testreglar/254/Nett/nett-2.5.4a.json

```diff
@@ -0,0 +1,151 @@
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har virksomheten svart at nettstedet har funksjonalitet som kan betjenes med bevegelsesaktivering?",
+			"ht": "<p>Sjekk dokumentasjonen mottatt fra virksomheten. </p>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke funksjonalitet med bevegelsesaktivering."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken funksjonalitet tester du?",
+			"ht": "<p>Bruk informasjonen fra virksomheten. Beskriv funksjonaliteten, slik at det er mulig å identifisere den i ettertid. Hvis det gjelder flere funksjoner, registrerer du en og en.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			},
+			"label": "Funksjonalitet:",
+			"datalist": "Sideutvalg"
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Hvilken type bevegelsesaktivering har funksjonaliteten?",
+			"ht": "<p>Kravet gjelder for følgende:</p>\n<ul>\n<li>bevegelser med selve enheten, gjennom for eksempel å vippe, rotere eller riste på den</li>\n<li>brukerbevegelser mot enheten, for eksempel ved at brukeren gjør ulike gester mot kamera </li>\n</ul>\n<p>Bruk informasjonen virksomheten og registrer bevegelsen. Eksempler på bevegelsaktivering er:</p>\n<ul>\n<li>Brukeren snur på enheten for å gå til neste eller forrige side</li>\n<li>Brukeren beveger enheten rundt for å se på ulike deler av et bilde</li>\n<li>Brukeren lukker en nettside ved å riste på enheten</li>\n<li>Brukeren gir tommel opp mot kamera for å godkjenne en handling</li>\n</ul>",
+			"type": "radio",
+			"kilde": [],
+			"svarArray": [
+				"Bevege selve enheten",
+				"Brukerbevegelser (gester) mot enheten"
+			],
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er bevegelsesaktivering avgjørende for funksjonaliteten?",
+			"ht": "<p>Åpne testsiden på smarttelefon eller nettbrett. </p>\n<p>Noen ganger er nettinnhold avhengig av å bruke enhetssensordata eller bevegelsen er kontrollert av nettleseren eller operativsystemet. Da er bevegelsen avgjørende for funksjonaliteten. </p>\n<p>Eksempler på innhold som er unntatt fordi det er avgjørende vil være når:</p>\n<ul>\n<li>operativsystemet på enheten har funksjonalitet for å \n<ul>\n<li>slette innskrevet tekst ved å riste på den</li>\n<li>slå av lyden på innkommende anrop ved å snu skjermen ned</li>\n<li>dra håndflaten over skjermen for å ta skjermbilde</li>\n</ul>\n</li>\n<li>en skritteller er avhengig av enhetens bevegelser for å registrere og telle skritt</li>\n</ul>",
```

Comment:
Jeg vil mene at hvis funktionaliteten kommer fra operativsystem, browser eller hjælpemiddelteknologi, så er det ikke in-scope for en WCAG-evaluering.

Eksemplet med "en skritteller er avhengig av enhetens bevegelser for å registrere og telle skritt" er efter min mening et eksempel på en "essential" funktion.
Et andet eksempel kunne være at spille tennis i et spil på skærmen ved at slå med sin hånd. 
De andre eksempler kunne så vidt jeg kan se udføres lige så godt vha. en brugergrænsefladekontrol.